ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -65,10 +65,14 @@ literals:
   - OFF
 - discharge_fan_speed_mode:
   - AUTO
+  - AUTOLOW
```

Comment:
LOW MEDIUM and HIGH are already defined. Remove AUTO.

Reply:
In total, there are 8 modes for the Taiwan discharge_fan_speed_mode:
<html>
<body>
<!--StartFragment--><google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}--></style>

0 | OFF
-- | --
1 | HIGH
2 | MEDIUM
3 | LOW
4 | AUTOHIGH
5 | AUTOMEDIUM
6 | AUTOLOW
7 | VFD

<!--EndFragment-->
</body>
</html>

Reply:
I remember the type, but I am unclear why we have more than 1 state for HIGH. Is there a difference in how it operates in AUTOHIGH and HIGH? Isn't there another field on the controller (`control_mode`, e.g.) that tells us if it is auto or manual?

Reply:
They are different. The run_mode is the entire device operation. While the discharge_fan_speed_mode is only applied for the discharge fan. This means when the run_mode is manual, people could still switch the discharge_fan_speed_mode to AUTOLOW, AUTOMEDIUM, etc.

Reply:
please explain how LOW and AUTOLOW are different. your last comment confuses me... how could the device be in manual mode but set to autolow?

Reply:
understood on run_mode: {AUTO, MANUAL}. 

the speed is still confusing me. is it fair to say that the difference between AUTO[SPEED] and [SPEED] is that the former is an automatically chosen setting by the controller (based on zone conditions) and the latter is manually set? That is, in the top screenshot is that fan in MED mode indefinitely, or can the controller automatically change it to LOW or HIGH?

I do not understand the functional difference between AUTOLOW/AUTOMED/AUTOHIGH and LOW/MED/HIGH; they seem to do the same thing. What am I missing?

Reply:
I have gotten the explanation from the vendor. There is no difference between LOW and AUTOLOW, MEDIUM and AUTOMEDIUM, HIGH AND AUTOHIGH. Due to the equipment contains these modes, so these modes must be included.

Reply:
@charbull @trav3711 @berkoben if we end up with multiple states that mean the same thing, can they all map to the same state? for example:
LOW: low
LOW: autolow
MED: med
MED: automed

etc.

Is there anything illegal about doing this?

Reply:
@tasodorff , 

the following is the current config for discharge_fan_speed_mode:
    discharge_fan_speed_mode:
      present_value: "points.discharge_fan_speed_mode.present_value"
      states:
        AUTOLOW: "6"
        AUTOHIGH: "4"
        HIGH: "1"
        MEDIUM: "2"
        LOW: "3"
        VFD: "7"
        AUTOMEDIUM: "5"
        "OFF": "0"
-------------------
I believe it is illegal to have a duplicate key in yaml e.g. 
   discharge_fan_speed_mode:
      present_value: "points.discharge_fan_speed_mode.present_value"
      states:
        LOW: "6"
        HIGH: "4"
        HIGH: "1"
        MEDIUM: "2"
        LOW: "3"
        VFD: "7"
        MEDIUM: "5"
        "OFF": "0"

Reply:
@charbull @trav3711 @berkoben please review

Reply:
@conroy-chee what do you think about an array?
LOW:
  -  "3"
   - "6"
High: 
   - "4" 
   - "1"
Off : "0"

Reply:
@charbull @tasodorff , this map is good. I have already made changes according to such mapping. I will drop the AUTO* mode for discharge_fan_speed_mode.

    discharge_fan_speed_mode:
      present_value: "points.discharge_fan_speed_mode.present_value"
      states:
        HIGH:
        - "1"
        - "4"
        MEDIUM:
        - "2"
        - "5"
        LOW:
        - "3"
        - "6"
        VFD: "7"
        "OFF": "0"

Reply:
@charbull i assume this is legal in instance validator? If it is not, i will open an issue for having it modified.